### PR TITLE
Features/goodbye share count

### PR DESCRIPTION
**Pull request by daaain**
Have a look @collingo, I'm particularly proud of the commit [Yo Dawg, I herd you like flexbox, so I put a flexbox in your flexbox so you can flexbox while you flexbox](https://github.com/ustwo/ustwo.com-frontend/commit/1f6dd6c0b9e16112314a55e04648e233fc817e32) (with a little help from @phil-linnell of course :hand:).

Yo Dawg

### CODE REVIEW

**Comment by phil-linnell** [comment]
Think @daaain wins 'commit message of the year'! We should to have an award ceremony :+1:

**Comment by daaain** [comment]
Hear hear @phil-linnell :) :fireworks:

File: src/app/flux/store.js [hunk 1]
```diff
@@ -263,7 +198,6 @@ const Store = Object.assign(
           type: 'posts'
         }], applyMorePosts).then(() => {
           Store.emit('change', _state);
```
Comment (collingo) [inline src/app/flux/store.js]: Niggly one but I'm just remembering, before I added in the `getSocialSharesForPost` call this line was in the condensed arrow function form...
```
}], applyMorePosts).then(() => Store.emit('change', _state));
```
Might be nice to restore that :)

File: src/server/bootstrapper.js [hunk 2]
```diff
@@ -100,15 +92,7 @@ function bootstrapper(initialUrl, hostApi, proxyUrl) {
...
-      let result;
-      if(some(itemsToLoad, item => item.type === 'posts')) {
-        result = getSocialSharesForPosts();
-      } else if(some(itemsToLoad, item => item.type === 'post')) {
-        result = getSocialSharesForPost();
-      } else {
-        result = _state;
-      }
-      return result;
+      return _state;
```
Comment (collingo) [inline src/server/bootstrapper.js]: This line would definitely benefit from the condensed arrow function form...
```
.then(() => _state);
```

**Comment by collingo** [comment]
Just a couple of little ones but otherwise all grand!

**Pull request merged**